CHANGELOG.md

```diff
@@ -175,20 +211,23 @@ included on the command line.
 More details about these changes can be found on our GitHub repo:
 https://github.com/certbot/certbot/issues?q=is%3Aissue+milestone%3A0.9.0+is%3Aclosed
 
-# 0.8.1
-## 06/14/2016
+## 0.8.1 - 2016-06-14
+
+### Improved
 
-* Preserving a certificate's common name when using `renew`
+* Preserve a certificate's common name when using `renew`
 * Save webroot values for renewal when they are entered interactively
-* Problems with an invalid user-agent string on OS X
-* Gracefully reporting the Apache plugin isn't usable when Augeas is not installed
+* Gracefully report the Apache plugin isn't usable when Augeas is not installed
 * Experimental support for Mageia has been added to `certbot-auto`
 
+### Fixed
+
+* Fix problems with an invalid user-agent string on OS X
```

Comment:
I think this is fixing a bug where the macos user agent string was causing an issue with our code. But yes a bug was fixed